lib/webauthn/encoder.rb

```diff
@@ -1,55 +1,112 @@
-# frozen_string_literal: true
-
 require "base64"
 
 module WebAuthn
-  def self.standard_encoder
-    @standard_encoder ||= Encoder.new
+  module Config
+    module Encoder
+      # Default encoding type used for WebAuthn operations
+      DEFAULT_ENCODING = :base64url
+
+      # Supported encoding types and their corresponding Base64 methods
+      ENCODINGS = {
+        base64: :strict,
+        base64url: :urlsafe
+      }.freeze
```

Comment:
We are not using the values of this hash right? Why not just storing the encodings as an array?